Source/WTF/wtf/Vector.h

```diff
@@ -55,6 +55,8 @@ namespace WTF {
 DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(Vector);
 DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(VectorBuffer);
 
+enum class NulloptBehavior : bool { Ignore, Abort };
```

Comment:
Annoying that we have to add this but I think we do! Or maybe we could use a `std::variant` with a type other than `std::nullopt_t` for the abort behavior.

Should we make ignore the default?

Reply:
I liked the idea of returning a std::optional since this is what we do for `compactMap()` for example. I feel that using `std::variant` would add some complexity. However, I think it would make sense to make "Ignore" the default.